imapserver/conn.go

```diff
@@ -85,6 +85,13 @@ func (c *Conn) Bye(text string) error {
 	return closeErr
 }
 
+func (c *Conn) IsEnabled(cap imap.Cap) bool {
```

Comment:
What is the purpose of exporting this method?

(If it's useful, maybe we should export an `EnabledCaps` method instead which would cover more use-cases… But need to be careful around locks.)

Reply:
>This can be used by backend implementations to disable/enable (potentially expensive) RECENT handling if IMAP4rev2 is in use.

Reply:
I'd prefer to either leave it for later, or return the whole cap set.

Reply:
I changed it to EnabledCaps method that creates a copy of enabled set and returns it.

Reply:
FWIW, the client side uses a copy-on-write approach (we can add this optimization later).

Reply:
As an server optimization, it does not really seem to be necessary to call EnabledCaps more than once after first SELECT as clients are not allowed to issue ENABLE after it:
> Clients MUST NOT issue ENABLE once they SELECT/EXAMINE a mailbox
(RFC 5161)